packages/@lwc/node-reactions/src/types.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+export type ReactionType = 1 /* connected */ | 2 /* disconnected */;
```

Comment:
Any reason why we didn't use [enums](https://www.typescriptlang.org/docs/handbook/enums.html) here?
```
export enum ReactionType {
  Connected,
  Disconnected,
  None,
}
```

Reply:
We did for perf. If we used enum, it would add a `.` property lookup to the execution path. In the original PR, I started out with an enum. After discussing with Caridy, we moved to this specific flags. So we sacrificed code readability for perf.

Reply:
I agree it would be great to avoid the performance penalty. What about exporting constants instead and using those constants instead of number literals?

```js
export const REACTION_TYPE_CONNECTED = 1;
export const REACTION_TYPE_DISCONNECTED = 2;
```

Terser does constant folding and will inline the references automatically for you.